### PR TITLE
Add quick new task function

### PR DESCRIPTION
**Pull request by janfer95**
Attempts to solve #29. 

To indicate that everything went smoothly I just echo "Task created successfully" after the query, not sure if this is the best way to approach it.

### CODE REVIEW

File: todoist.el [hunk 1]
```diff
@@ -335,6 +335,21 @@ P is a prefix argument to select a project."
...
+  "Create a new task without loading the todoist-buffer.
+
+CONTENT is the content string.
+DUE is the human friendly due string and can be empty.
+P is a prefix argument to select a project."
+  (interactive "sTask content: \nsDue: \nP")
+  (todoist--query "POST" "/tasks"
+                  (append `(("content" . ,content) ("due_string" . ,due))
+                          (when p
+                            `(("project_id" . ,(todoist--project-id (todoist--select-project)))))))
```
Comment (abrochard) [inline todoist.el]: We should avoid duplication. If this is a copy paste of lines 328-335, those lines should be exacted into a "private" function and shared by both `todoist-new-task` and `todoist-quick-new-task`.

**Comment by abrochard** [comment]
Thank you for the PR! I do like having a separate function for "quick new task" but I think it's best to avoid code duplication. I am gonna try to take a pass at it asap but I am swamped these days. I left a comment for now.
I do like the echo "Task created successfully". Any sort of feedback to the user is good.

File: todoist.el (hunk 1 above)
Comment (janfer95) [reply todoist.el]: Yes, it was simply a copy paste of those lines. I do agree that a private function is cleaner. Should be better now!

**Comment by janfer95** [comment]
No worries, I also only now managed to get back to it! I created the private function todoist--create-new-task to avoid duplication. When you have time you can check if it's better now (and if it also works on your machine).

**Review by abrochard — APPROVED** [review]
Looks good! Thank you

**Pull request merged**